FBRetainCycleDetector/Graph/FBObjectGraphConfiguration.h

```diff
@@ -49,7 +51,14 @@ typedef FBGraphEdgeType (^FBGraphEdgeFilterBlock)(FBObjectiveCGraphElement *_Nul
  */
 @property (nonatomic, readonly) BOOL shouldInspectTimers;
 
+/**
+ Will cache layout
+ */
+@property (nonatomic, readonly, nullable) NSMutableDictionary<Class, NSArray<id<FBObjectReference>> *> *layoutCache;
+@property (nonatomic, readonly) BOOL shouldCacheLayouts;
+
 - (nonnull instancetype)initWithFilterBlocks:(nonnull NSArray<FBGraphEdgeFilterBlock> *)filterBlocks
-                         shouldInspectTimers:(BOOL)shouldInspectTimers NS_DESIGNATED_INITIALIZER;
+                         shouldInspectTimers:(BOOL)shouldInspectTimers
+                           shouldCacheLayout:(BOOL)shouldCacheLayout NS_DESIGNATED_INITIALIZER;
```

Comment:
Why make it adjustable? The tests are passing `NO`, but does caching even effect the tests?

Is there a reason someone would create a `FBObjectGraphConfiguration` with no caching?